tests/phpunit/tests/theme/wpGetBlockCssSelector.php

```diff
@@ -0,0 +1,445 @@
+<?php
+
+require_once __DIR__ . '/base.php';
```

Comment:
What is this?

Reply:
Theme [setup scripts](https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/theme/base.php).

Not needed here. Removed in [e1c3210](https://github.com/WordPress/wordpress-develop/pull/4655/commits/e1c3210b3a2b4870cad3aad937b1444c739f4fc3)